### DISCUSSION TITLE
GitHub Lab is not continuing, I would like to run it as an OSS

### DISCUSSION THREAD

**Discussion by wildcard** [community/community#24105] [GitHub Education]
<p>Please GitHub can you share GitHub lab code as open source so we can host it publicly for everyone that wants to still run the platform</p>

**Comment by wildcard** [comment]
<p>I would like to add that GitHub Skills is not feature complete in comparison to GitHub Lab. Having GitHub Lab code available as Open source will allow us to keep our internal Education operations going &amp; help other users to host their own labs too.</p>
<p>In addition, GitHub Lab yaml structure is effective for us to organize our tutorials &amp; we would like to keep use this structure. We already integrated it with some of our internal systems.</p>

**Comment by jfullstackdev** [comment]
<p>try posting it here</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github-community/community">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github-community/community" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47092464/181082704-baf8bac9-8e7c-4ea2-97f8-4f6ded9f277c.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/4/2401d7bd09718eb3e29d860a1ec5a5c99a969161_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/4/2401d7bd09718eb3e29d860a1ec5a5c99a969161_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/4/2401d7bd09718eb3e29d860a1ec5a5c99a969161.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/4/2401d7bd09718eb3e29d860a1ec5a5c99a969161_2_10x10.png"></div>

<h3><a href="https://github.com/github-community/community" target="_blank" rel="noopener">GitHub - github-community/community: Public feedback discussions for: GitHub...</a></h3>

  <p>Public feedback discussions for: GitHub for Mobile, GitHub Discussions, GitHub Codespaces, GitHub Sponsors, GitHub Issues and more! - GitHub - github-community/community: Public feedback discussion...</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>that is the Prod Team,</p>

**Comment by wildcard** [comment]
<p>I did <a href="https://github.com/github-community/community/discussions/19875#discussion-4188636" class="inline-onebox" rel="noopener nofollow ugc">GitHub Lab is not continuing, I would like to run it as an OSS · Discussion #19875 · github-community/community · GitHub</a></p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Thanks for reaching out, we’re glad to hear you enjoyed using Learning Lab. We do not have plans to open-source this product as part of the deprecation process.</p>

**Comment by wildcard** [comment]
I've created a migration tool over GitHub actions you're welcome to use it https://github.com/marketplace/actions/trasform-github-lab-to-github-skills-format
